src/foam/nanos/approval/ApprovalRequest.js

```diff
@@ -46,7 +47,8 @@
     'ctrl',
     'currentMenu',
     'stack',
-    'user'
+    'user',
```

Comment:
'subject' please, unless you some other part of approval is exporting 'user'.

Reply:
Removed. Wasn't even used in the first place.